cmd/copyDownloadBlobFSEnumerator.go

```diff
@@ -30,6 +30,7 @@ func (e *copyDownloadBlobFSEnumerator) enumerate(cca *cookedCopyCmdArgs) error {
 
 	// attempt to parse the source url
 	sourceURL, err := url.Parse(cca.source)
+	sourceURL = gCopyUtil.appendQueryParamToUrl(sourceURL, cca.sourceSAS)
```

Comment:
Looks odd with this new line inserted between the Parse call and the if statement that checks the parse result.  Better to put this after the if that checks the parse error.  Oh, and I see there's a comment on gCopyUtil deprecating its use. Maybe use the local util (which is already defined above, and does the same thing).